app/src/main/res/values/strings.xml

```diff
@@ -43,6 +43,11 @@
     <string name="close_text">CLOSE</string>
     <string name="logout_text">LOG OUT</string>
 
+    <string name="default_time_homepage">00</string>
```

Comment:
Let's make sure it says 00:00:00 at the end of the event this year haha

Reply:
lol yes